data/companies.yml

```
@@ -292,6 +292,13 @@ pro_services:
       - "InfraCloud is your end-to-end Istio service mesh consulting & implementation partner - whether you're just starting out or want to take it to the next level."
       - "The team of service mesh consultants assesses your microservice landscape and proposes the right strategy and roadmap for your goals and deploy it successfully. Plus, get hands-on experience and training with InfraCloud team by your side enabling you to make the most of your Istio deployment."
       - "InfraCloud also offers enterprise-grade Istio service mesh experts, whenever and wherever you need them to resolve your production issues. The service mesh experts can help you optimize, migrate to Istio or build custom features for your specific needs."
+  - name: "Cloud Scale® Inc"
+    logo: "/logos/cloudscaleinc.png"
+    url: "https://cloudscaleinc.com/"
+    description: "Cloud Scale® Inc experts assess your current applications, runtime deployments, service maps and use-cases, find the right mix of cloud platforms for your workloads, identify gaps in technology & skills, and prepare a success plan – The ones that will help you accomplish your business objectives."
```

```suggestion
    description: "Cloud Scale® Inc experts assess your current applications, runtime deployments, service maps and use-cases, find the right mix of cloud platforms for your workloads, identify gaps in technology & skills, and prepare a success plan."
```

I think it's too much longer than the others.  If this LGTM, I can merge.